### PR TITLE
Revert "Fetch api, display data, style home and detail pages"

### PR DESCRIPTION
**Pull request by BasitKorede**
# Fetch Api
This pull request does the following 

- display a list of countries and filter them by name
- fetch API, display data from API
- create test
- style home , and detail page

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,206 +1,6 @@
...
-  justify-content: space-between;
-  align-items: center;
-  font-size: 1.2rem;
-}
-
-.details-row:nth-child(odd) {
-  background-color: var(--darkpurple);
-}
-
-.details-header {
```
Comment (alexander16108) [inline src/index.css:202]: from what I can see all your files have reverted and hold no current changes to the project, please fix this and request a  review again

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @BasitKorede , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.css (hunk 1 above)
Comment (BasitKorede) [reply src/index.css:202]: Hello @alexander16108 thanks for the review, I really appreciate your feedback. I mistakenly merge the pull request I was supposed to submit that was why I submitted the reverted one. My question now is that show I make a new commit and open a pull request then submit it or what do you think I should do ?

**Comment by BasitKorede** [comment]
# Note to reviewer:
I Mistakenly merged the initial branch I am supposed to submit so this is the reverted one . Kindly check the merged one for the file changes.
I am very sorry 😔 for the unnecessary work it is a mistake which will not repeat itself.
Thanks for your understanding @BasitKorede .

**Review by FrederickMih — COMMENTED** [review]
Hi @BasitKorede , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that is not working properly. This is because you didn't submit the correct working branch. Please submit the PR with the correct working branch and leave a link to the merge PR and another link to the PR in which the previous reviewer has left a comment that is the link to this PR. You can equally commit the latest changes to this branch.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**